app/build.gradle

```diff
@@ -20,7 +20,7 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled true
+            minifyEnabled false
```

Comment:
// Enables code shrinking, obfuscation, and optimization for only
 // your project's release build type.
// Please change it to true